### DISCUSSION TITLE
Wrong data from qrz.com

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#380] [General]
I tried to do a lookup for my call from qrz.com (log QSO with myself). But lookup returns partially wrong data: LoTW membership, grid square and map position.

![изображение](https://github.com/wavelog/wavelog/assets/16667173/dfbc8da8-0b82-4128-ad91-2ee24cef98fb)

**Comment by phl0** [comment]
LoTW membership is not retrieved from qrz.com but from LoTW (which is down atm). The grid (KP50da) is correctly fetched from qrz.com here. Do you use qrz.com or hamqth.com for lookup here?

**Reply by R1BLH** [reply]
It's understandable about LotW,
I'm using lookup from qrz.com. How can I debug this situation?

**Reply by R1BLH** [reply]
Lookup from hamqth.com works perfectly:

![изображение](https://github.com/wavelog/wavelog/assets/16667173/6027d42e-3bd8-4e61-b41c-d6c82702efc6)

**Reply by TomJohnson0358** [reply]
This confuses me.  The wiki states that HamQTH is not supported, but then gives directions on how to update config.php to use HamQTH.  There are no places in the UI to enter HamQTH, as far as I can tell.

**Reply by phl0** [reply]
> This confuses me. The wiki states that HamQTH is not supported, but then gives directions on how to update config.php to use HamQTH. There are no places in the UI to enter HamQTH, as far as I can tell.

Lookup for HamQTH is configured in config.php and this is supported. Will find the Wiki page and correct.

**Reply by TomJohnson0358** [reply]
Ah, thank you.  I understand now.
